crypto/err/openssl.txt

```diff
@@ -843,8 +843,8 @@ EVP_F_EVP_KEYEXCH_FETCH:245:EVP_KEYEXCH_fetch
 EVP_F_EVP_KEYEXCH_FROM_DISPATCH:244:evp_keyexch_from_dispatch
 EVP_F_EVP_MAC_CTRL:209:EVP_MAC_ctrl
 EVP_F_EVP_MAC_CTRL_STR:210:EVP_MAC_ctrl_str
-EVP_F_EVP_MAC_CTX_DUP:211:EVP_MAC_CTX_dup
-EVP_F_EVP_MAC_CTX_NEW:213:EVP_MAC_CTX_new
+EVP_F_EVP_MAC_CTX_DUP:211:EVP_MAC_dup_ctx
+EVP_F_EVP_MAC_CTX_NEW:213:EVP_MAC_new_ctx
```

Comment:
Considering that you removed them in code, feel free to just remove these lines.